tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/RecordSession.cs

```diff
@@ -100,7 +102,19 @@ public RecordEntry Lookup(RecordEntry requestEntry, RecordMatcher matcher, IEnum
         {
             foreach (RecordedTestSanitizer sanitizer in sanitizers)
             {
+                RecordEntry reqEntryPreSanitize = null;
+                if (DebugLogger.CheckLogLevel(LogLevel.Debug))
+                {
+                    requestEntry.initializeIsModifiedFlag();
```

Comment:
If we do these in the deserialization property, we shouldn't need to do it here.

Reply:
Leaving it as is, as discussed offline.